SU-HSDO/suhumsci
# fix(shs-5112): fixing contrast on ext vertical links

# READY FOR REVIEW

## Summary
The PR adds a small CSS fix to ensure that links that are external and used in vertical cards display properly in various color schemes. 

## Urgency
medium

## Steps to Test
1. Visit a page like: https://sgs.suhumsci.loc/academics/undergraduate-programs that uses external links in the vertical card component
2. Ensure that they are receiving vertical linked title theming (and not generic ext link theming.) If they are using the correct theming, they will match cards that are using internal links (i.e., all the cards will match.) 

Example URLS:
https://creativewriting.stanford.edu/undergraduate/opportunities-undergraduate-writers
https://sgs.stanford.edu/academics/undergraduate-programs

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
